apps/server/src/external/demo/app.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * This is a very minimal example for a websocket client
+ * You could use this as a starting point to creating your own interfaces
+ *
+ * This example does not handle disconnections
+ */
+/* eslint-disable */
```

Comment:
why do we need to disable eslint?

Reply:
there are missing defines
I found this `/*eslint-env browser*/` seams better?

Reply:
I am not sure I understood. Can you send me a screenshot?

Reply:
with out it default browser defines are marked as missing
![image](https://github.com/cpvalente/ontime/assets/19875125/c167b407-9fa1-406e-8d63-d825eac0fcc6)

Reply:
I wouldnt worry about eslint here
One better change could be checking if websocket is available in the environment